### PR TITLE
Portfolio website with advanced layout and styling

### PR DESCRIPTION
**Pull request by melsylv**

### CODE REVIEW

File: melodyPortfolio.html [hunk 1]
```diff
@@ -21,15 +21,17 @@ <h1>Melody Sylvestre</h1>
...
 				<div class="aboutMe_text">
-					<p>A nice little paragraph where I talk about myself. Bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla. Bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla</p>
+					<p>Hi! I'm Melody! I recently graduated from IO Academy (Bath) as a full stack software developer. In a previous life, I worked as a planetary scientist in France and in the UK for 9 years.
+						
+			 <p>I'm curious and I love a good challenge, especially if it gives me an opportunity to get outside of my comfort zone! Outside work, I enjoy singing, playing piano, sewing, crochet and all things space-related!</p>	
 				 <a href="images/cv_msylvestre.pdf" target="_blank">Download my CV</a>
 			 </div>
```
Comment (hamya5) [inline melodyPortfolio.html:36]: Need to tidy up indentations from here

File: style.css [hunk 2]
```diff
@@ -46,23 +62,24 @@ nav a {
...
+	margin-right: 300px;
 }
 
 .introduction_section p {
-	background-color: black;
+	background-color: rgba(0,0,0,0.5);
 	color: white;
-	margin-right: 50px;
-	font-size: 25px;
+	margin-right: 300px;
```
Comment (hamya5) [inline style.css:80]: add margin-left? 
on laptop screen size text butts against left side of page

File: style.css [hunk 3]
```diff
@@ -71,38 +88,63 @@ nav a {
...
+.portfolio_section {
+	background-color: #171616;
+/*  	padding: 50px 50px;*/
+	padding-top: 50px;
+	padding-bottom: 50px;
+}
+
 .portfolio_section h2 {
-	margin-left: 50px;
+	margin: 0;
```
Comment (hamya5) [inline style.css:147]: also some margin-left? 
or padding-left in .portfolio_section?
same as with introduction text...

**Review by hamya5 — CHANGES_REQUESTED** [review]
Most comments apply to wider screen stylings. Issues aren't there on mobile size.

Navbar: consider making it fixed while scrolling.

Intro section: on laptop size, text is right against left side of screen.

About section: on wider screen, phot and text layout is a bit jumbled. Maybe bring paragraphs closer together and put space between text and photo.
Also consider making photo edges less sharp, perhaps just a little border-radius?

Portfolio section: heading placement needs improving; against left side of page and too close to first project card.

Contact section: consider making text more "noticeable".
Also, on wide screen the icons seem very far apart. 
Whole section feels a bit squashed.

Whole page: full white text on full black background feels a bit sharp. Consider using slightly grey shade for text.

File: melodyPortfolio.html (hunk 1 above)
Comment (melsylv) [reply melodyPortfolio.html:36]: Done

File: style.css (hunk 2 above)
Comment (melsylv) [reply style.css:80]: done!

File: style.css (hunk 3 above)
Comment (melsylv) [reply style.css:147]: done

**Review by hamya5 — APPROVED** [review]
Nickel!

**Pull request merged**